sleap/io/format/nix.py

```diff
@@ -0,0 +1,270 @@
+
+    So far the adaptor exports the tracked positions for each node of each instance, the track and skeleton information along with the respective scores and the centroid. Additionally, the video information is exported as metadata.
+    For more information on the mapping from sleap to nix see the docs on [nixtrack](https://github.com/bendalab/nixtrack) (work in progress).
+    The adaptor uses a chunked writing approach which avoids numpy out of memory exceptions when exporting large datasets.
+
+    author: Jan Grewe (jan.grewe@g-node.org)
+    """
+    @property
+    def default_ext(self):
+        return "nix"
+
+    @property
+    def all_exts(self) -> List[str]:
+        return [self.default_ext]
+
+    @property
+    def handles(self):
+        return SleapObjectType.misc
+
+    @property
+    def name(self) -> str:
+        """Human-reading name of the file format"""
+        return "NIX file flavoured for animal tracking data https://github.com/g-node/nix"
+
+    @classmethod
+    def can_read_file(file: FileHandle) -> bool:
+        """Returns whether this adaptor can read this file."""
+        return False
+
+    @classmethod
+    def can_write_filename(filename: str) -> bool:
+        """Returns whether this adaptor can write format of this filename."""
+        raise NotImplementedError
+
+    @classmethod
+    def does_read() -> bool:
+        """Returns whether this adaptor supports reading."""
+        return False
+
+    @classmethod
+    def does_write() -> bool:
+        """Returns whether this adaptor supports writing."""
+        return nix_available
+
+    @classmethod
+    def read(file: FileHandle) -> object:
+        """Reads the file and returns the appropriate deserialized object."""
+        raise NotImplementedError("NixAdaptor does not support reading.")
+
+    @classmethod
+    def write(cls, filename: str, source_object: object, source_path: str = None, video: Video = None):
+        """Writes the object to a file."""
+        def check(labels, video):
+            if video is None and len(labels.videos) == 0:
+                return False
+            return True
+
+        def create_file(filename, project, video):
+            nf = nix.File.open(filename, nix.FileMode.Overwrite)
+            s = nf.create_section("TrackingAnalysis", "nix.tracking.metadata")
+            s["version"] = "0.1.0"
+            s["format"] = "nix.tracking"
+            s["definitions"] = "github.com/bendalab/nix_tracking"
+            s["writer"] = str(cls)[8:-2]
+            if project is not None:
+                s["project"] = project
+            if video is not None:
+                name = os.path.split(video.backend.filename)[-1]
```

Comment:
We're working on converting over to `pathlib` instead of `os`:
```suggestion
                name = Path(video.backend.filename).name
```